ts/Core/Renderer/HTML/AST.ts

```diff
@@ -20,28 +20,59 @@ import type HTMLAttributes from '../HTML/HTMLAttributes';
 import type SVGAttributes from '../SVG/SVGAttributes';
 
 import H from '../../Globals.js';
-const { SVG_NS } = H;
+const {
+    SVG_NS,
+    win
+} = H;
 import U from '../../Utilities.js';
 const {
     attr,
     createElement,
-    discardElement,
     error,
+    isFunction,
     isString,
     objectEach,
     splat
 } = U;
+const {
+    trustedTypes
+} = win;
 
 /* *
  *
  *  Constants
  *
  * */
 
+// Create the trusted type policy. This should be encapsuled in the module.
+const trustedTypesPolicy = (
+    trustedTypes &&
+    isFunction(trustedTypes.createPolicy) &&
+    trustedTypes.createPolicy(
+        'highcharts', {
+            createHTML: (s: string): string => s
+        }
+    )
+);
+
+// Create a TrustedHTML object from an untrusted string. This should also not be
+// exposed, and only used in our controlled cases (empty string and prior to
+// running the DOMparser).
+const trustedHTML = function (html: string): string {
```

Comment:
The only problem I have here it's about the `trustedHTML` name. It might return an unsafe string. I guess that's the moment where DOMParser comes in handy?

Reply:
> we would need to check in other places

True, but only in two places - when creating the empty HTML, and before running `parseFromString`. I refactored this, got rid of the `trustedHTML` function.

Reply:
Nice, thanks!